go2rtc_client/rest.py

```diff
@@ -21,7 +21,8 @@
 _LOGGER = logging.getLogger(__name__)
 
 _API_PREFIX = "/api"
-_SUPPORTED_VERSION: Final = AwesomeVersion("1.9.4")
+_MIN_VERSION_SUPPORTED: Final = AwesomeVersion("1.9.4")
+_MAX_EXCLUDED_VERSION: Final = AwesomeVersion("2.0.0")
```

Comment:
This name is not good, it's not the highest unsupported version, it's the lowest unsupported version